roles/user_config/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+# purpose : adds support for creating SASL users and ACLs
```

Comment:
Need a no_log: true on every place a password could be referenced and exposed in a debug log